source/crud/write-operations/bulk.txt

```diff
@@ -358,6 +359,33 @@ see the following API documentation:
 - `BulkWriteOptions <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/BulkWriteOptions.html>`__
 - `ordered() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/BulkWriteOptions.html#ordered(boolean)>`__
 
+
+BulkWrite Example: Full File
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following code is a complete, standalone file that performs an ordered bulk
+write operation. 
+
+.. include:: /includes/crud/example-intro.rst
+
+This example call to ``bulkWrite()`` includes examples of the ``InsertOneModel``,
+``UpdateOneModel``, ``DeleteOneModel``, and ``ReplaceOneModel``. 
```

Comment:
S: similar to my suggestion for the find example, I'd introduce the code example with the list, something like "This code example is a complete, standalone file that performs the following actions:

- Creates an InsertOneModel, UpdateOneModel ... to specify the write operations
- Runs an ordered bulk write operation to perform the specified writes